electrode-ota-server-routes-acquisition/src/index.js

```diff
@@ -85,7 +85,10 @@ export const register = diregister({
                     download(request.params.packageHash, (e, o) => {
                         if (e) return reply(e);
                         const { content, length } = o;
-                        reply(content).type("application/octet-stream").bytes(length);
+                        reply(content)
+                            .header("Cache-Control", "s-max-age=31536000, max-age=0, must-revalidate")
```

Comment:
why these types?

Reply:
to have a 1day of Cache n revalidate again ?

Reply:
`s-max-age` is mispelled. get rid of `must-revalidate`